tests/test_config.py

```diff
@@ -258,3 +258,12 @@ def test_athena_wait_delay_config_override(wr: ModuleType, glue_database: str) -
         mock_wait_query.assert_called_once()
 
         assert mock_wait_query.call_args[1]["athena_query_wait_polling_delay"] == polling_delay_argument
+
+
+def test_load_from_env_variable(wr: ModuleType) -> None:
+    s3_endpoint_url = "s3.us-east-1.amazonaws.com"
```

Comment:
Nitpick, but perhaps we should avoid making this test specific to `us-east-1`? Either a regex or we choose a different env variable to test?